chain/network/src/config_json.rs

```diff
@@ -44,6 +44,12 @@ fn default_peer_stats_period() -> Duration {
     Duration::from_secs(5)
 }
 
+// If true - we'll skip sending tombstones during initial sync.
+fn default_skip_tombstones() -> bool {
+    // Enable by default in shardnet only.
+    cfg!(feature = "shardnet")
```

Comment:
I assume that you actually want the flags to be settable via the JSON config. Because if it was not the case,
in network config we have a "features" field which is set independently from the JSON config in runtime and it can depend on the chain_id for example.

Reply:
yeah - I want it to be 'settable' - so that we can change it 'dynamically' if needed.

Once we finish the rollout of the timestamp-based nonces, we can cleanup this code.